src/main/java/org/generationcp/middleware/dao/dms/PhenotypeDao.java

```diff
@@ -942,14 +943,17 @@ public Boolean containsAtLeast2CommonEntriesWithValues(final int projectId, fina
 				observationUnit.setStudyDbId((String) row[8]);
 				observationUnit.setStudyName((String) row[9]);
 				observationUnit.setProgramName((String) row[10]);
-				String x = (String) row[16]; // ROW
-				String y = (String) row[17]; // COL
+
+				String x = row[16] != null ? (String) row[16] : null; // ROW
+				String y = row[17] != null ? (String) row[17] : null; // COL
 				if (StringUtils.isBlank(x) || StringUtils.isBlank(y)) {
-					x = (String) row[11]; // fieldMapRow
-					y = (String) row[12]; // fieldMapCol
+					x = row[11] != null ? (String) row[11] : "1"; // fieldMapRow
+					y = row[12] != null ? (String) row[12] : "1"; // fieldMapCol
```

Comment:
`1, 1` for all :thinking:

Reply:
This also got me thinking. If we set these values to null, KSU fieldbook will fail to import a study, because the app expects these fields to have values, and it also uses these fields as secondary and tertiary IDs. Maybe we can change these to 0 as default?

Reply:
Mmm I see. I guess it's right to say that if there's no layout, then everything is in one "big" cell call `(1,1)`. `0` might work too but I don't know how Field book will take it. And `0` is problematic in some places. Yeah, let's keep `(1, 1)`